draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -1211,26 +1198,27 @@ Here are some interoperability issues:
   * X-RateLimit-Remaining and X-Rate-Limit-Remaining
   * X-RateLimit-Reset and X-Rate-Limit-Reset
 
+The semantic of RateLimit depends on the windowing algorithm.
 The semantic of RateLimit depends on the windowing algorithm.
 A sliding window policy for example may result in having a
 remaining keyword
 value related to the ratio between the current and the maximum throughput.
 e.g.
 
 ~~~ example
-RateLimit: limit=12,    \
-           remaining=6, \ ; using 50% of throughput, that is 6 units/s
-           reset=1
+RateLimit-Limit: 12
 RateLimit-Policy: 12;w=1
+RateLimit-Remaining: 6          ; using 50% of throughput, that is 6 units/s
+RateLimit-Reset: 1
 ~~~
 
 If this is the case, the optimal solution is to achieve
 
 ~~~ example
-RateLimit: limit=12,   \
-           remaining=1 \  ; using 100% of throughput, that is 12 units/s
-           reset=1
+RateLimit-Limit: 12
 RateLimit-Policy: 12;w=1
+RateLimit-Remaining: 1          ; using 100% of throughput, that is 12 units/s
+RateLimit-Reset: 1
```

Comment:
Why did these get reverted to the older style?

Reply:
An error. I will fix.